### ISSUE TITLE
White space above headerbar on mobile

### ISSUE THREAD

**Issue by ryanleesipes** [thundernest/thunderblog#11]
![Screenshot_20220223-124303.png](https://user-images.githubusercontent.com/2141621/155294673-73dfcf29-0cf1-4777-97f6-92f72670aeac.png)

See above. There is white space (just a thin line) above the header bar unless you are actively scrolling down on mobile.

That shouldn't be there. Should be reproducible by scrolling up or upon page load on mobile.

**Comment by devmount** [comment]
@ryanleesipes Thanks for reporting. Which OS/browser/version are you using?

I tested with Chrome, Opera and Firefox on Android 10 and simulated some more devices, but wasn't able to reproduce this yet.

**Comment by ryanleesipes** [comment]
Thank you. Tagging @Sancus to see if he can reproduce.

**Comment by devmount** [comment]
Ok I took another look at this and found, that this might be an issue of the Firefox mobile app. I was able to reproduce this with _Scroll to hide toolbar_ enabled and scrolling the page down and up again. Disabling this option removes the thin background colored line:

![image](https://user-images.githubusercontent.com/5441654/159082587-2e79c88e-f960-44e5-b5b0-852bf44edca5.png)

I also played around with CSS positioning, but CSS seems to have no effect here, since this seems to be some kind of scrolling glitch. In my opinion, this isn't a Thunderblog bug.
